solver/src/settlement_simulation.rs

```diff
@@ -113,6 +118,7 @@ mod tests {
             &web3,
             network_id.as_str(),
             Block::FixedWithTenderly(block),
+            0.0,
```

Comment:
I am surprised that this works. But I guess for simulation against the test env the gas price is not relevant.